### ISSUE TITLE
Device compatibility: oat path

### ISSUE THREAD

**Issue by schrnz** [Project-ARTist/ArtistGui#18]
We mainly work on Nexus and Pixel devices where oat files are stored in the same location. Other vendors might have changed the folder structure, hence we should test whether ArtistGui works there of if we need to add other oat folder alternatives or even a routine to actively search for the oat path.

**Comment by sweisgerber-dev** [comment]
Shouldn't we wait until complain's arrive regarding this? We don't have any sufficient number of different devices to test this.
Also, it's pretty unlikely that the oat-file location changed and the architecture subfolder already gets probed (`saarland.cispa.artist.artistgui.utils.AndroidUtils#probeArchitetureFolderName`):

- /data/app/<package-name>/oat/arm
- /data/app/<package-name>/oat/arm64

**Comment by schrnz** [comment]
I agree that we currently do not have the capacity and devices to test a broad range. However, I was already told that for one or two manufacturers, the path is different.

Let's keep this issue around with low priority. As soon as actual issues surface, we can have a look at services like firebase that enable testing on a multitude of devices.
